lib/autocomplete-manager.coffee

```diff
@@ -110,26 +113,59 @@ class AutocompleteManager
     return if @isCurrentFileBlackListed()
     cursor = @editor.getLastCursor()
     return unless cursor?
-    cursorPosition = cursor.getBufferPosition()
-    currentScope = cursor.getScopeDescriptor()
-    return unless currentScope?
-    currentScopeChain = currentScope.getScopeChain()
-    return unless currentScopeChain?
-
-    options =
-      editor: @editor
-      buffer: @buffer
-      cursor: cursor
```

Comment:
If we did ever do that we could pass the cursor to the provider's `getPrefix` method, no? Or we could pass the position. I really dont see the need for this method, though.